torch_frame/data/multi_tensor.py

```diff
@@ -325,6 +325,37 @@ def select(
     def _single_index_select(self, index: int, dim: int) -> _MultiTensor:
         raise NotImplementedError
 
+    def fillna(self, fill_value: Union[float, int]) -> None:
```

Comment:
I think it would help us to use PyTorch notation for in-place ops, e.g., in this case, this should be `fillna_`.

Reply:
Got it.